webpack.config.js

```diff
@@ -357,7 +358,9 @@ if (!DEV) {
 const env = {};
 if (DEV) {
     env.PUBLIC_PATH = JSON.stringify(publicPath);
-} else {
+    if (process.env.MM_LIVE_RELOAD) { //eslint-disable-line no-process-env
+        config.plugins.push(new LiveReloadPlugin());
+    }
     env.NODE_ENV = JSON.stringify('production');
```

Comment:
This is changing entirely the meaning of this line, I would expect to maintain that in the `else` right?

Reply:
Doh. A typo, sorry. 🤦‍♂️

Reply:
No problem, for this are the Peer reviews :)